src/cdk-experimental/clipboard/clipboard.md

```diff
@@ -0,0 +1,62 @@
+**Warning: this component is still experimental. It may have bugs and the API may change at any
+time**
+
+The scrolling package provides helpers for working with the system clipboard.
+
+## The `Clipboard` service
+
+The `Clipboard` service copies text to the
+user's clipboard. It has two methods, `copy` and `beginCopy`. For cases where
+you are copying a relatively small amount of text, you can just call `copy` to
+place it on the clipboard.
+
+```typescript
+class MyCopier {
+  constructor(private clipboard: Clipboard) {}
+
+  copy() {
+    this.clipboard.copy('Hello clipboard!');
+  }
+}
+```
+
+However, for a large amount of text the browser needs time to load it into a
```

Comment:
"load it into a hidden textarea where it can be copied" -> "fill an intermediate textarea element and copy the content"

Reply:
Done.